packages/cxl-ui/src/components/cxl-app-layout.js

```diff
@@ -1,13 +1,14 @@
 /**
  * @todo implement primary action button.
  */
-import { LitElement, html, customElement, property, query } from 'lit-element';
 import '@conversionxl/cxl-lumo-styles';
 import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
-import cxlAppLayoutStyles from '../styles/cxl-app-layout-css.js';
-import cxlAppLayoutGlobalStyles from '../styles/global/cxl-app-layout-css.js';
 import '@vaadin/vaadin-button';
 import '@vaadin/vaadin-context-menu/src/vaadin-device-detector.js';
+import { customElement, html, LitElement, property, query } from 'lit-element';
+import normalizeWheel from 'normalize-wheel';
+import cxlAppLayoutStyles from '../styles/cxl-app-layout-css.js';
+import cxlAppLayoutGlobalStyles from '../styles/global/cxl-app-layout-css.js';
```

Comment:
Did ESLint or something auto-re-arrange these?

Ideally we'd only have the relevant import change here.

Reply:
That's a habit of mine: https://code.visualstudio.com/docs/languages/typescript#_organize-imports

I can change it back, here's the relevant eslint rule if we decide to enable it in the future: https://eslint.org/docs/rules/sort-imports

Reply:
Yes, I would prefer stylistic changes to be done in a separate commit with a single linter run for everything.

Sort Imports rule makes sense, that could be a separate PR first for `master`, and we'll verify whether everything stays working. There's a couple of places I know that are specific about their import order.

Reply:
Fixed